doc/develop/test/bsim.rst

```diff
@@ -76,3 +76,103 @@ Check :ref:`the page on coverage generation <coverage_posix>` for more info.
 
 .. _EDTT:
    https://github.com/EDTTool/EDTT
+
+Tests with radio activity
+*************************
+
+The ``zephyr/tests/bsim/`` folder contains tests meant to be run with BabbleSim's physical layer
+simulation, and therefore cannot be run directly from twister.
+
+The ``compile.sh`` and ``run_parallel.sh`` scripts contained in that folder are used by the CI system to
+build the needed images and execute these tests in batch.
+
+Building and running
+====================
+
+See the :ref:`nrf52_bsim` page for setting up the simulator.
+
+The scripts also expect a few environment variables to be set.
+For example, from Zephyr's root folder, you can run:
+
+.. code-block:: bash
+
+   # Build all the tests
+   WORK_DIR=${ZEPHYR_BASE}/bsim_out ${ZEPHYR_BASE}/tests/bsim/compile.sh
```

Comment:
Sidenote: Setting `WORK_DIR` is optional nowadays (it defaults to `${ZEPHYR_BASE}/bsim_out`)

Reply:
I removed it, its less verbose.